### PR TITLE
Don't include _url in ddoc 

### PR DESCRIPTION
**Pull request by mandric**
# Description

https://github.com/medic/medic-projects/issues/2473

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: Changes.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+## 2.8.6
+
+### TBD
+
+#### Continuous Integration
+
+- Automatically update to master branches of `api` and `sentinel` submodules
+  before running CI in Travis. [#3621](https://github.com/medic/medic-webapp/pull/3621)
+- Fixed kanso packages that inadvertantly cached data. Issue: #3648
+
```
Comment (garethbowen) [inline Changes.md]: The first item isn't included in this release - remove it.

The second item isn't a CI issue, it's a security one, so change the header to Security.

**Comment by mandric** [comment]
Weird error in CI:
https://travis-ci.org/medic/medic-webapp/builds/262024706#L3528
```
07 08 2017 22:36:49.841:WARN [Firefox 52.0.0 (Linux 0.0.0)]: Disconnected (1 times), because no message in 60000 ms.
Firefox 52.0.0 (Linux 0.0.0): Executed 221 of 518 DISCONNECTED (1 min 16.363 secs / 1.48 secs)
Warning: Task "karma:unit_ci" failed. Use --force to continue.
Aborted due to warnings.
```

File: Changes.md (hunk 1 above)
Comment (mandric) [inline Changes.md]: The first one actually is, but in the commit before this.

Comment (garethbowen) [inline Changes.md]: Ok well in that case: I don't think CI fixes are relevant in the changelog as the audience doesn't care. It doesn't change the code they get. So I think remove it.

Comment (mandric) [inline Changes.md]: Ok I don't mind either way.  But I don't think the audience  is only app consumers.

**Comment by mandric** [comment]
Ok changes are in, Gareth.

**Pull request merged**